### DISCUSSION TITLE
Added Feature Request for JetBrains IDEs

### DISCUSSION THREAD

**Discussion by SommerEngineering** [nushell/nushell#12661] [Show and tell]
I love Nu shell ❤️. I would also like to use the shell in my JetBrains IDEs together with the "new terminal" ([link](https://blog.jetbrains.com/idea/2024/02/the-new-terminal-beta-is-now-in-jetbrains-ides/)). Therefore, I have created a feature request at JetBrains: [link](https://youtrack.jetbrains.com/issue/IJPL-148780). Are you also interested? Then please upvote the request 👍.

**Comment by fdncred** [comment]
I think it would be cool to be supported in the JetBrains IDE. I'm not sure what it would take on our side though.
